gui/public/i18n/zh-Hans/translation.ftl

```diff
@@ -216,6 +216,12 @@ tracker-status-disconnected = 断开连接
 tracker-status-occluded = 被遮挡
 tracker-status-ok = 已连接
 tracker-status-timed_out = 连接超时
+imu-not-applicable = 不匹配的传感器
```

Comment:
Translation files besides english shouldn't be modified directly, they're managed by Pontoon at https://i18n.slimevr.dev